commands/command_integrate.go

```diff
@@ -0,0 +1,206 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"os/user"
+	"path/filepath"
+	"strings"
+
+	"github.com/github/git-lfs/vendor/_nuts/github.com/spf13/pflag"
+
+	"github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra"
```

Comment:
Yeah, my Sublime setup runs `goimports` every time I save, it obviously doesn't strips blanks.